### PR TITLE
implement beverage image picker

### PR DESCRIPTION
**Pull request by AntonPuko**
close https://github.com/Brewskey/Brewskey.App/issues/192
close https://github.com/Brewskey/Brewskey.App/issues/118

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -81,7 +81,9 @@ class App extends React.Component<{}> {
       }
 
       if (AuthStore.isAuthorized) {
-        NavigationService.navigate('main');
+        NavigationService.navigate('editBeverage', {
+          id: '76419',
```
Comment (jlkalberer) [inline src/App.js]: hmmm

File: src/components/BeverageImagePicker.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import type { EntityID } from 'brewskey.js-api';
+import type { Style } from '../types';
+
+import * as React from 'react';
+import * as ImagePicker from 'react-native-image-picker';
+import { observer } from 'mobx-react/native';
+import BeverageAvatar from '../common/avatars/BeverageAvatar';
+
+const IMAGE_PICKER_OPTIONS = {
+  title: 'Select beverage sticker',
```
Comment (jlkalberer) [inline src/components/BeverageImagePicker.js]: nit: 'Select beverage photo'

File: src/screens/EditBeverageScreen.js [hunk 3]
```diff
@@ -38,9 +40,22 @@ class EditBeverageScreen extends InjectedComponent<InjectedProps> {
...
+    const id = nullthrows(values.id);
+
+    DAOApi.BeverageDAO.put(id, beverageMutator);
     await waitForLoaded(() => this._beverageLoader);
+    if (beverageImage) {
+      await waitForLoaded(() =>
+        UpdateBeverageImageStore.get(id, beverageImage),
+      );
+      UpdateBeverageImageStore.flushCache();
+      flushImageCache(`https://brewskey.com/cdn/beverages/${id.toString()}`);
```
Comment (jlkalberer) [inline src/screens/EditBeverageScreen.js]: Isn't the `brewskey.com` in a variable somewhere?

File: src/screens/NewBeverageScreen.js [hunk 4]
```diff
@@ -21,10 +23,22 @@ type InjectedProps = {|
...
+    const { beverageImage, ...beverageMutator } = values;
+    const clientID = DAOApi.BeverageDAO.post(beverageMutator);
     const { id } = await waitForLoaded(() => BeverageStore.getByID(clientID));
+
+    if (beverageImage) {
+      await waitForLoaded(() =>
+        UpdateBeverageImageStore.get(id, beverageImage),
+      );
+      UpdateBeverageImageStore.flushCache();
+      flushImageCache(`https://brewskey.com/cdn/beverages/${id.toString()}`);
```
Comment (jlkalberer) [inline src/screens/NewBeverageScreen.js]: Same

File: src/App.js (hunk 1 above)
Comment (AntonPuko) [reply src/App.js]: ops

**Pull request merged**